src/order/zorn.lean

```diff
@@ -272,6 +272,12 @@ begin
         (λ ht, h t₁ ht₁ c₁ hc₁ c₂ (ht hc₂) hneq) }
 end
 
+lemma chain.empty : chain ∅ :=
+chain_chain_closure chain_closure_empty
+
+lemma chain.subsingleton (hc : set.subsingleton c) : chain c :=
+λ _ hx _ hy hne, (hne (hc hx hy)).elim
```

Comment:
I don't think the `.` in the name is useful here
```suggestion
lemma chain_empty : chain ∅ :=
chain_chain_closure chain_closure_empty

lemma chain_subsingleton (hc : set.subsingleton c) : chain c :=
λ _ hx _ hy hne, (hne (hc hx hy)).elim
```
unless the rest of the `chain` api already follows the pattern you have.